localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_draenei_1;Elekk Pasture;;Elekk-Weide;;;;;;;;;;;x
+ca_culture_draenei_2;Elekk Pasture;;Elekk-Weide;;;;;;;;;;;x
+ca_culture_draenei_3;Elekk Pasture;;Elekk-Weide;;;;;;;;;;;x
+ca_culture_draenei_4;Elekk Pasture;;Elekk-Weide;;;;;;;;;;;x
+ca_culture_draenei_desc;Upon their arrival on Draenor, the draenei found herds of gentle, elephant-like creatures known as elekks. Succeeding in domesticating them, the draenei developed such a bond with the species that they brought elekks with them on their escape from Outland. Elekks are smart, loyal, large, patient and able to cover long distances without tiring, making them ideal mounts, whether for general travel or to act as heavy cavalry.;;Bei ihrer Ankunft auf Draenor fanden die Draenei Herden von sanften, elefanten�hnlichen Kreaturen vor, die als Elekks bekannt sind. Da es den Draenei gelang, sie zu domestizieren, entwickelten sie eine solche Bindung zu dieser Art, dass sie auf ihrer Flucht aus der Scherbenwelt auch Elekks mitbrachten. Elekks sind klug, treu, gro�, geduldig und in der Lage, gro�e Entfernungen zu �berwinden, ohne zu erm�den, was sie zu idealen Reittieren macht, sei es f�r allgemeine Reisen oder als schwere Kavallerie.;;;;;;;;;;;x
+ca_culture_murloc_1;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_2;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_3;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_4;Murloc Elemental Totems;;Murloc-Elementartotems;;;;;;;;;;;x
+ca_culture_murloc_desc;Though the magic of murlocs is relatively primitive compared to other shamanic cultures, murlockind shows a great aptitude and connection to water and earth elemental magics. The magic may be tame and fairly basic, but is terrifying in power when murloc casters come together in great numbers, supported by their fanatic, spear-swinging, followers.;;Obwohl die Magie der Murlocs im Vergleich zu anderen schamanischen Kulturen relativ primitiv ist, zeigt die Art der Murlocs eine gro�e Begabung und Verbundenheit mit der Elementarmagie des Wassers und der Erde. Die Magie mag zahm und recht simpel wirken, aber ihre Macht ist erschreckend, wenn Murloc-Wirker in gro�er Zahl zusammenkommen, unterst�tzt von ihren fanatischen, speerschwingenden Anh�ngern.;;;;;;;;;;;x
```

Comment:
> murlockind

Genuinely not sure about this. Is this a word that is actually used? If not, it could just be changed to `they show`. Left it for now, though.
In addition, unnecessary comma was removed in the last sentence.
```suggestion
ca_culture_murloc_desc;Though the magic of murlocs is relatively primitive compared to other shamanic cultures, murlockind shows a great aptitude and connection to water and earth elemental magics. The magic may be tame and fairly basic, but is terrifying in power when murloc casters come together in great numbers, supported by their fanatic, spear-swinging followers.;;Obwohl die Magie der Murlocs im Vergleich zu anderen schamanischen Kulturen relativ primitiv ist, zeigt die Art der Murlocs eine gro�e Begabung und Verbundenheit mit der Elementarmagie des Wassers und der Erde. Die Magie mag zahm und recht simpel wirken, aber ihre Macht ist erschreckend, wenn Murloc-Wirker in gro�er Zahl zusammenkommen, unterst�tzt von ihren fanatischen, speerschwingenden Anh�ngern.;;;;;;;;;;;x
```

Reply:
oh, didn't see that comma ;)